### DISCUSSION TITLE
JUNO非齐次泊松过程的疑问

### DISCUSSION THREAD

**Discussion by hyy-hyy-19309** [physics-data/faq#136] [Q&A] [answered]
可以问一下卷积的积分上下限吗？如果是负无穷到正无穷的话不就是一个指数衰减的函数吗？
![image](https://user-images.githubusercontent.com/80196186/132124715-f63365c4-55a2-469f-96d6-4bc55945504c.png)
另外那个高斯分布是关于什么的高斯分布？σ的单位是什么？

**Comment by heroxbd** [comment]
@Berrysoft

**Comment by Berrysoft** [comment, marked as answer]
指数的定义域为0到正无穷，因此卷积相当于一个分段函数与一个正态分布PDF的卷积：

![MommyTalk1630903638920](https://user-images.githubusercontent.com/37586447/132162075-84451a1c-19d3-49e6-a8e5-91b20b5fb2fa.png)

其中

![MommyTalk1630903737199](https://user-images.githubusercontent.com/37586447/132162179-aaf2679b-0480-465d-ac17-0514cfd9d841.png)
